File: app/src/main/java/ml/dvnlabs/animize/database/model/userland.java
package ml.dvnlabs.animize.database.model;

public class userland {
    public static final String table_name = "userland";
    public static final String col_id_user = "id_user";
    public static final String col_name_user = "name_user";
    public static final String col_access_token = "token";
    public static final String col_email = "email";

    private String id;
    private String token;
    private String name;
    private String email;

    public static final String CREATE_TABLE = "CREATE TABLE " +table_name+"("+col_id_user+" TEXT, "+
            col_name_user + " TEXT, "+col_email+" TEXT, "+col_access_token+" TEXT)";

    public userland(){

    }
    public userland(String id_user,String name_user,String email,String tokeen){
        this.email = email;
        this.id = id_user;
        this.name = name_user;
        this.token = tokeen;

    }

    public String getNameUser() {
        return name;
    }

    public String getEmail() {
        return email;
    }

    public String getIdUser() {
        return id;
    }

    public String getToken() {
        return token;
    }

    public void setToken(String token) {
        this.token = token;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public void setNameUser(String name) {
        this.name = name;
    }

    public void setIdUser(String id) {
        this.id = id;
    }
}

File: app/src/main/java/ml/dvnlabs/animize/database/model/UserLand.kt
/*
 * Copyright (c) 2020.
 * Animize Devs
 * Copyright 2019 - 2020
 * Davin Alfarizky Putra Basudewa <dbasudewa@gmail.com>
 * This program used for watching anime without ads.
 *
 */
package ml.dvnlabs.animize.database.model

class UserLand(id_user: String?, name_user: String?, var email: String?, tokeen: String?) {
    var idUser: String? = id_user
    var token: String? = tokeen
    var nameUser: String? = name_user

    companion object {
        const val table_name = "userland"
        const val col_id_user = "id_user"
        const val col_name_user = "name_user"
        const val col_access_token = "token"
        const val col_email = "email"
        const val CREATE_TABLE = "CREATE TABLE " + table_name + "(" + col_id_user + " TEXT, " +
                col_name_user + " TEXT, " + col_email + " TEXT, " + col_access_token + " TEXT)"
    }
}